lib/cli/ui/formatter.rb

```diff
@@ -136,6 +137,20 @@ def parse_expr(sc, stack)
               index
             )
           end
+        elsif match = sc.scan(SCAN_WIDGET)
+          # We could call Regexp.last_match but that wouldn't be threadsafe.
```

Comment:
AFAIK `Regexp.last_match` is in fact thread-safe

Reply:
Oh wow, cool! Thanks.